src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -94,13 +96,20 @@
 ;; TEXT
 (defn render-block
   [{:keys [content content-type edited-at in-popover?]} acc
-   {:keys [type ^js literal children]}]
+   {:keys [type ^js literal children]}
```

Comment:
Just a side note, not that you need to change anything.

We could remove the type hint on the `literal` argument. It's really odd to add a type hint to a string in Clojure, because Clojure already provides lots of functions to process strings, so why would we use the Javascript ones? 

```clojure
;; This is unnecessary.
(.substring literal 0 (.-length literal))

;; Good
(subs literal 0 (count literal))
```